src/utils/reduceSQON.ts

```diff
@@ -34,6 +34,11 @@ const reduceSQON = (sqon: SQON): SQON => {
 			op: sqon.op,
 			content: [],
 		};
+		if (sqon.pivot !== undefined) {
```

Comment:
```suggestion
		if (sqon.pivot) {
```

Reply:
If they set `sqon.pivot=false` then it won't be propagated this way. That may be functionally equivalent but its not the same.